.eslintrc

```diff
@@ -1,31 +1,34 @@
 {
-    "parser": "babel-eslint",
-    "extends": [
-        "airbnb",
-        "prettier",
-        "prettier/flowtype",
-        "prettier/react",
-        "plugin:jest/recommended"
+  "parser": "babel-eslint",
+  "extends": [
+    "airbnb",
+    "prettier",
+    "prettier/flowtype",
+    "prettier/react",
+    "plugin:jest/recommended"
+  ],
+  "plugins": ["emotion"],
+  "rules": {
+    "emotion/jsx-import": "error",
+    "react/jsx-filename-extension": "off",
+    "react/require-default-props": "off",
+    "jsx-a11y/anchor-is-valid": "off",
```

Comment:
Can we pass options as suggested here:

https://github.com/evcohen/eslint-plugin-jsx-a11y/issues/402

instead of turning this rule off?

Reply:
Tracked as AFP-238